levelkdev/swapr-dapp
# Fix/1047 migrate to arbitrum/sdk

# Summary

Fixes [#1047](https://github.com/levelkdev/swapr-dapp/issues/1047)

Migrate arb-ts to arbitrum/sdk

  # To Test
This work affects mostly the bridge functionalities.
To test, perform different operations that were previously working in the Bridge.
1. Check if `Arbitrum` displays correct bridge details like `fee`, `toAmount`
2. Make sure that `validate` method is working as expected
3. Do a transaction from `l1Chain` (`Ethereum` or `Rinkeby`) to `l2Chain` (`Arbitrum One` or `Arbitrum Rinkeby`)
4. Do a transaction from `l2Chain` to `l1Chain`
5. Try to `approve` token
6. Try to execute collect method
7. Check transaction status on `ui`


  # Background

arb-ts is deprecated. Although it works, migrating to the current dependency will remove one future concern from the project.

Done in colaboration with Daniel and mix1o.
